apps/settings/js/modules/storage/default_media_volume.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * DefaultMediaVolume:
+ *   - DefaultMediaVolume is an Observable that wraps the platform
+ *     DeviceStorage object.
+ *   - DefaultMediaVolume is a singleton that you can easily use it to fetch
+ *     some shared data across different panels.
+ *   - It has some observable properties: currentVolume.
+ * DefaultMediaVolume only update state and does not involve in any UI logic.
+ *
+ * @module DefaultMediaVolume
+ */
+define(function(require) {
+  'use strict';
+
+  var DeviceStorages = require('modules/storage/device_storages');
+  var Observable = require('modules/mvvm/observable');
```

Comment:
`Observable` now provides a new way for defining a new class of it. It extends prototype definition and is able to define read only, dependency, and ordinary properties. You may refer to [[1]] for details.
[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/operator_settings/models/operator_item.js